src/components/content-box/_content-box.scss

```diff
@@ -34,7 +34,7 @@ $include-html: false !default;
 
     &--spaced {
       .mint-content-box__content, .mint-content-box__footer {
-        @media (min-width: $breakpointMedium) {
+        @include mintBreakpoint('medium-up') {
```

Comment:
Do you think single quotes + `-` are better than no quotes and `_` (`medium_up`) or even `mediumUp`